nixos/modules/services/audio/tts.nix

```diff
@@ -106,7 +106,7 @@ in
             + optionalString (options.useCuda) " --use_cuda"
             + (concatMapStringsSep " " escapeShellArgs options.extraArgs);
           CapabilityBoundingSet = "";
-          DeviceAllow = if options.useCuda then [
+          DeviceAllow = optionalString options.useCuda [
```

Comment:
Should this be `optionals` instead? `optionalString` works because dynamically-typed language, but... I don't know, is it worth pretending `optionalString` only wants strings?

Reply:
This changed the hardening properties of the module. An empty string means we make sure no devices are allowed, while an empty list gets dropped from the unit entirely, thus setting no empty DeviceAllow list.

It needs to be list and fallback to string otherwise.

Reply:
I reverted my changes in this file.